src/rendering/text/TextView.ts

```diff
@@ -99,14 +108,7 @@ export class TextView implements View
 
         this._style?.off('update', this.onUpdate, this);
 
-        if (style instanceof TextStyle)
-        {
-            this._style = style;
-        }
-        else
-        {
-            this._style = new TextStyle(style);
-        }
+        this._style = ensureTextStyle(this._renderMode, style);
```

Comment:
for `style` we are using `TextStyle` as the type, however that isn't true anymore. We should be using `TextStyle | HTMLTextStyle` everywhere